packages/suite/src/types/utils/index.ts

```diff
@@ -38,3 +40,8 @@ export type DeepPartial<T> = T extends () => any
     : T extends { [key: string]: any }
     ? { [P in keyof T]?: DeepPartial<T[P]> }
     : T;
+
+// Type the yup schema shape
```

Comment:
I don't like it is placed in this file, I know this is types file, but it makes much more sense to have this type placed in `packages/suite/src/config/suite/yup.ts` to have all related things in one place. It will also save us time during imports because these type will be used everytime when yup will be used

Reply:
agree. done